sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -125,8 +106,23 @@ public void sendChatReadReceipt(String chatThreadId, SendReadReceiptRequest send
      * @throws CommunicationErrorResponseException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
      */
-    public void sendChatMessage(String chatThreadId, SendChatMessageRequest sendChatMessageRequest, final Callback<SendChatMessageResult> callback) {
-        this.serviceClient.sendChatMessage(chatThreadId, sendChatMessageRequest, callback);
+    public void sendChatMessage(String chatThreadId, SendChatMessageRequest sendChatMessageRequest, final Callback<String> callback) {
+        Callback<SendChatMessageResult> proxyCallback = new Callback<SendChatMessageResult>() {
+            @Override
+            public void onSuccess(SendChatMessageResult result, okhttp3.Response response) {
+                if (result == null) {
```

Comment:
When result is null, isn't is a failure? If yes, should we call onFailure instead?